sos/report/plugins/ceph_osd.py

```diff
@@ -63,34 +98,8 @@ def setup(self):
                 "ceph-volume lvm list"
             ])
 
-            cmds = [
-               "bluestore bluefs available",
-               "config diff",
-               "config show",
-               "dump_blacklist",
-               "dump_blocked_ops",
-               "dump_historic_ops_by_duration",
-               "dump_historic_slow_ops",
-               "dump_mempools",
-               "dump_ops_in_flight",
-               "dump_op_pq_state",
-               "dump_osd_network",
-               "dump_reservations",
-               "dump_watchers",
-               "log dump",
-               "perf dump",
-               "perf histogram dump",
-               "objecter_requests",
-               "ops",
-               "status",
-               "version",
-            ]
-
-            self.add_cmd_output(
-              [f"ceph daemon {i} {c}" for i in self.get_socks() for c in cmds]
-            )
-
         else:
+            directory = '/var/snap/microceph'
             # Only collect microceph files, don't run any commands
             self.add_forbidden_path([
                 "/var/snap/microceph/common/**/*keyring*",
```

Comment:
same as above with `directory` variable replacement?

Reply:
Same reasons as above, leaving as it is